dvc/output/base.py

```diff
@@ -195,17 +196,35 @@ def get_hash(self):
                 self.path_info,
                 self.fs,
                 self.fs.PARAM_CHECKSUM,
+                dvcignore=self.dvcignore,
             ).hash_info
         return ostage(
-            self.odb, self.path_info, self.fs, self.odb.fs.PARAM_CHECKSUM
+            self.odb,
+            self.path_info,
+            self.fs,
+            self.odb.fs.PARAM_CHECKSUM,
+            dvcignore=self.dvcignore,
         ).hash_info
 
     @property
     def is_dir_checksum(self):
         return self.hash_info.isdir
 
+    def is_ignored(self, path) -> bool:
```

Comment:
I understand that later we need to use it to check `abs_path` but `is_ignored` method in context of `output` does not seem to make sense. Output is a concept of some singular asset and methods like `exists` `is_ignored` are regarding the output itself. I think that `is_ignored` should be implemented for `self.path_info` and in case of `_validate_output_paths` we should be using `dvcignore` object

Reply:
@pared , Yes, and so I renamed it here.